trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -155,7 +155,9 @@ Rather, the entry would be rewritten to only include the most recent position:
 `congo=congosSecondPosition,rojo=rojosFirstPosition`
 
 **Limits:**
-Maximum length of a combined header MUST be less than 512 bytes. 
+Maximum length of a combined header MUST be less than 512 bytes.
+
+`tracestate` field contains essential information for requests correlation. Platforms and tracing systems MUST propagate this header. Compliance with specification will require storing of `tracestate` as part of request payload or associated metadata. Allowing the long field values can make compliance to the specification impossible. Thus the aggressive limit of 512 bytes was chosen.
```

Comment:
I still think a more concrete example of why 512 vs 2048 or any other fixed val

Reply:
The history of 512 was that it was a number nobody wanted to go below. I heard another suggestion is to haver total of `tracestate` and `traceparent` be under 1K.

CC: @bogdandrutu who has opinion and example with some queue service and metadata.

Reply:
I would rephrase this part to be a bit more open. I suggest we define this as:

Tracestate must be forwared by all participating tracing if it i is less or equal than 512 bytes. If the tracestate is bigger than this, the tracer can decide to forward the trace state or drop it.

Note: I know we talked about truncation. However, since there is no final decision on how this would work, the current working draft should state that you either forward or drop.

Reply:
👍 on the wording by @AloisReitbauer